spec/image_scraper/client_spec.rb

```diff
@@ -18,12 +18,16 @@
 
   describe "#initialize" do
     it 'works with invalid URLs' do
+      allow_any_instance_of(described_class).to receive(:fetch).and_return(nil)
+
       scraper = described_class.new('bogusurl4444.com')
```

Comment:
without this the test takes 10 seconds trying to hit a url that doesn't actually exists